add aside reminding users they can't script chrome://extensions (#1166)
GoogleChrome/developer.chrome.com: site/en/docs/extensions/mv2/getstarted/index.md

click the button to turn it green! However, some users may want to change the background to a
different color.

{% Aside 'gotchas' %}

> You can't script any internal Chrome page, like "chrome://extensions". Be sure to try out the

Reviewer: It isn't obvious what "script a page" means. Can you rephrase?

> You can't run a content script on an internal Chrome page, such as "chrome://extensions". Be sure
> to try out the extension on a real webpage, like https://google.com.

Reviewer 2: ```suggestion
Extensions can not inject content scripts on internal Chrome pages like "chrome://extensions". Be
sure to try out the extension on a real webpage like [https://google.com](https://google.com).
```

Rephrase slightly to emphasize that this is a platform restriction, not an issue with the reader.

> You can't run a content script on an internal Chrome page, such as "chrome://extensions". Be sure
> to try out the extension on a real webpage, like https://google.com.

Reviewer 2: ```suggestion
Extensions can not inject content scripts on internal Chrome pages like "chrome://extensions". Be
sure to try out the extension on a real webpage like [https://google.com](https://google.com).
```

Rephrase slightly to emphasise that this is a platform restriction, not an issue with the reader.
